src/handlers/swap_quote_handlers.ts

```diff
@@ -0,0 +1,198 @@
+import { ExtensionContractType, Orderbook, SwapQuoteConsumer, SwapQuoter } from '@0x/asset-swapper';
+import { MarketOperation, SwapQuoterError } from '@0x/asset-swapper/lib/src/types';
+import { SupportedProvider } from '@0x/order-utils';
+import { BigNumber, RevertError } from '@0x/utils';
+import * as express from 'express';
```

Comment:
newline above?

Reply:
Not according to our linter or prettier?

Reply:
This looks fine?